### ISSUE TITLE
Cli - Provide option to set cluster for  "project create" and project for "rancher ps" using project name/cluster name

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#11469]
rancher cli  version - v1.0.0-alpha5

Currently  cluster for  "project create" and project for "rancher ps"  gets set from the default project set for the user during login.

We need to provide an option to set cluster for  "project create" and project for "rancher ps"  commands , in case where the user has logged in when he had no clusters or in cases were he would like to work with a cluster other than the default project.

**Comment by dramich** [comment]
Available in https://github.com/rancher/cli/releases/tag/v1.0.0-alpha8

**Comment by sangeethah** [comment]
@dramich Using "--cluster" option for  ```rancher create projects``` and "--project" option for ```rancher ps``` , able to switch context.

But both these options requires user to put in a clusterId/ projectId.

It would be more useful to have them be able to provide cluster name/project name instead.

**Comment by deniseschannon** [comment]
Related: https://github.com/rancher/rancher/issues/11459

**Comment by deniseschannon** [comment]
Cluster name and project name are not unique in a Ranchersetup so there's no way for CLI to use name.

**Comment by dramich** [comment]
Available in v1.0.0-alpha13

**Comment by sangeethah** [comment]
Tested with v1.0.0-alpha13

Able to create project in a different cluster (other than default) by specifying "--cluster" option to which cluster name/id can be passed.

Able to list workloads  from project (other than default) by specifying "--project" option to which cluster name/id can be passed.

```
 ./rancher project create --cluster santest1do --description "my new cluster" hello3
./rancher ps --project mynewp1
```
